doc/stdlib/strings.rst

```diff
@@ -113,6 +113,20 @@
 
    Returns true if the given char or integer is a valid Unicode code point.
 
+.. function:: isvalid(value) -> Bool
+
+   Returns true if the given value is valid for its type,
+   which currently can be one of Char, ASCIIString, UTF8String, UTF16String, or UTF32String
```

Comment:
OK, thanks again!  Should I also remove the documentation for the old `is_valid_xxx` functions?